config.js

```diff
@@ -673,6 +673,10 @@ var config = {
     */
     // dynamicBrandingUrl: '',
 
+
```

Comment:
Drop this empty line. We generally leave 1 empty live between settings, and 2 between "sections".